src/TreeSitter/TSNode.class.st

```diff
@@ -129,6 +129,13 @@ TSNode >> detectNamedChild: name ifNone: aConstantBlockClosure [
 	^ self collectNamedChild detect: [ :tsNode | tsNode type = name ] ifNone: aConstantBlockClosure
 ]
 
+{ #category : 'as yet unclassified' }
+TSNode >> detectNamedChild: nodeName ifOne: aBlock [ 
```

Comment:
We are trying to remove this "ifOne:" notation. It comes from VisualWorks IIRC but in Phaor we use "ifFound:" everywhere for detect.

Maybe it'd be better to rename it for "ifFound:" to stay coherent?

Reply:
yes :)